### PR TITLE
Migrate 4 components to Svelte

### PR DESCRIPTION
**Pull request by micheal-parks**
Migrates components:
* Motor
* Operations Sessions
* Movement sensor
* Servo

All tested locally with no issues.

### CODE REVIEW

File: web/frontend/src/components/servo/index.svelte [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  const req = new servoApi.StopRequest();
+  req.setName(name);
+
+  rcLogConditionally(req);
+  client.servoService.stop(req, new grpc.Metadata(), displayError);
+};
+
+const move = (amount: number) => {
+  const servo = rawStatus;
+  console.log(servo)
```
Comment (mcous) [inline web/frontend/src/components/servo/index.svelte:23]: Possible debugging leftover

File: web/frontend/src/components/motor/index.svelte [hunk 2]
```diff
@@ -0,0 +1,254 @@
...
+        <v-button
+          icon="play-circle-filled"
+          variant="success"
+          label="RUN"
+          on:click={motorRun}
+        />
+      </div>
+    </div>
+  </div>
+</v-collapse>
```
Comment (mcous) [inline web/frontend/src/components/motor/index.svelte:254]: Missing newline

**Review by mcous — APPROVED** [review]
Transfer LGTM, once that errant `console.log` is removed!

File: web/frontend/src/components/servo/index.svelte (hunk 1 above)
Comment (micheal-parks) [reply web/frontend/src/components/servo/index.svelte]: Fixed!

**Pull request merged**